pymc3/tests/test_step.py

```diff
@@ -113,3 +113,31 @@ def test_binomial(self):
             Binomial('x', 10, 0.5)
             steps = assign_step_methods(model, [])
         self.assertIsInstance(steps, Metropolis)
+
+class TestSampleEstimates(unittest.TestCase):
```

Comment:
if you run it with SeededTest, the values for the three medians (alpha, beta, sigma) are:
```
NUTS
[ 1.0461552   2.49790007]
1.00212348481
0.490281370723

Metropolis
[ 1.04715176  2.54407313]
0.993205192687
0.500511771013

Slice
[ 1.01062073  2.56799309]
1.02232315527
0.514094365842
```

on current master, the numbers are

```
NUTS
[ 1.04251656  2.48266657]
1.00953574057
0.505059014542

Metropolis
[ 1.05449217  2.49944286]
1.00122022558
0.502538412741

Slice
[ 1.05344913  2.44528951]
1.00594797036
0.503320463491
```

Seems hard to set a cutoff that catches the bad `NUTS` sampler, but not the `Slice`.

As an aside: the numbers are different because the samples are all being drawn in the same test case -- setting a random seed in the `sample` method makes them reproducible.  Does it make any sense to combine this with `TestStepMethods` ?  It looks like `check_stat` is doing a similar thing...